### PR TITLE
Fix BFT support

### PR DESCRIPTION
**Pull request by dzikowski**

### CODE REVIEW

File: src/setup-docker/templates/fabric-docker/commands-generated.sh [hunk 1]
```diff
@@ -15,10 +15,8 @@ generateArtifacts() {
   <%_ ordererGroups.forEach((ordererGroup) => { _%>
 
   <% if(!global.capabilities.isV3) {%> 
-  printItalics "Generating genesis block for group <%= ordererGroup.name %>" "U1F3E0"
-  genesisBlockCreate "$FABLO_NETWORK_ROOT/fabric-config" "$FABLO_NETWORK_ROOT/fabric-config/config" "<%= ordererGroup.profileName %>"
-  <% } else { %>
```
Comment (Sanket-0510) [inline src/setup-docker/templates/fabric-docker/commands-generated.sh:20]: This else condition was required correct? I feel we should have a log regarding system channel support

File: src/setup-docker/templates/fabric-docker/docker-compose.yaml [hunk 2]
```diff
@@ -188,29 +188,39 @@ services:
...
+      # TLS Configuration
       - ORDERER_GENERAL_TLS_ENABLED=true
       - ORDERER_GENERAL_TLS_PRIVATEKEY=/var/hyperledger/orderer/tls/server.key
       - ORDERER_GENERAL_TLS_CERTIFICATE=/var/hyperledger/orderer/tls/server.crt
       - ORDERER_GENERAL_TLS_ROOTCAS=[/var/hyperledger/orderer/tls/ca.crt]
+      # Cluster TLS Configuration
       - ORDERER_GENERAL_CLUSTER_CLIENTCERTIFICATE=/var/hyperledger/orderer/tls/server.crt
       - ORDERER_GENERAL_CLUSTER_CLIENTPRIVATEKEY=/var/hyperledger/orderer/tls/server.key
       - ORDERER_GENERAL_CLUSTER_ROOTCAS=[/var/hyperledger/orderer/tls/ca.crt]
-      - ORDERER_ADMIN_LISTENADDRESS=<%= orderer.address %>:7053
```
Comment (Sanket-0510) [inline src/setup-docker/templates/fabric-docker/docker-compose.yaml:212]: I am not sure but I feel for version 2 if TLS enabled we will need to specify this address

File: src/setup-docker/templates/fabric-docker/docker-compose.yaml [hunk 3]
```diff
@@ -188,29 +188,39 @@ services:
...
       <%_ } _%>
+      <%_ if(ordererGroup.consensus !== "BFT" && global.capabilities.isV2) { _%>
```
Comment (Sanket-0510) [inline src/setup-docker/templates/fabric-docker/docker-compose.yaml]: As per my information there is no support for BFT in version 2 so checking for consensus as BFT I feel is redundant here

File: src/setup-docker/templates/fabric-docker/commands-generated.sh (hunk 1 above)
Comment (dzikowski) [reply src/setup-docker/templates/fabric-docker/commands-generated.sh:20]: no, we don't need it. This is the default way Fabric works, no need to provide this information in logs

File: src/setup-docker/templates/fabric-docker/docker-compose.yaml (hunk 3 above)
Comment (dzikowski) [reply src/setup-docker/templates/fabric-docker/docker-compose.yaml:204]: correct, can be simplified

Comment (dzikowski) [reply src/setup-docker/templates/fabric-docker/docker-compose.yaml]: correct, can. be simplified

File: src/setup-docker/templates/fabric-docker/docker-compose.yaml (hunk 2 above)
Comment (dzikowski) [reply src/setup-docker/templates/fabric-docker/docker-compose.yaml:212]: I didn't verify v2 tests, that might be so - feel free to add also for v2 tls:

```
      # Admin endpoint configuration
      - ORDERER_ADMIN_TLS_ENABLED=true
      - ORDERER_ADMIN_TLS_CERTIFICATE=/var/hyperledger/orderer/tls/server.crt
      - ORDERER_ADMIN_TLS_PRIVATEKEY=/var/hyperledger/orderer/tls/server.key
      - ORDERER_ADMIN_TLS_ROOTCAS=[/var/hyperledger/orderer/tls/ca.crt]
      - ORDERER_ADMIN_TLS_CLIENTROOTCAS=[/var/hyperledger/orderer/tls/ca.crt]
      - ORDERER_ADMIN_LISTENADDRESS=0.0.0.0:<%= orderer.adminPort %>
```

File: src/setup-docker/templates/fabric-docker/docker-compose.yaml (hunk 3 above)
Comment (Sanket-0510) [reply src/setup-docker/templates/fabric-docker/docker-compose.yaml]:
```suggestion
      <%_ if(global.capabilities.isV2) { _%>
```

**Review by Sanket-0510 — APPROVED** [review]

**Pull request merged**